### PR TITLE
add groupid, tenant password options

### PR DESCRIPTION
**Pull request by protomech**
Add groupid, tenantPassword options for dsa_control.

### CODE REVIEW

**Comment by protomech** [comment]
@DeepSecurityOpenSourceSupport branch has been merged against latest master.

Please review; group id is important for our needs and may be useful for others.

File: tasks/activate-linux.yml [hunk 1]
```diff
@@ -66,6 +74,7 @@
     msg: 
         - "Activate DSA when it's not activated or forecd to reactivate."
         - "force_reactivation: {{ force_reactivation }}"
+        - "dsa_control_argument: {{ dsa_control_argument }}"
```
Comment (DeepSecurityOpenSourceSupport) [inline tasks/activate-linux.yml]: Please remove this as the argument might have security concern that we intent not to put it in msg.

**Comment by DeepSecurityOpenSourceSupport** [comment]
Thanks @protomech a lot for using our storybook and all the useful feedback, we're verifying this change and please help to do the following actions:
1. Remove the sensitive debug message as I mentioned in the change
2. Please help to REBASE with the master, we tend to keep the commit history clean without duplicated merge commits.
PS. Sorry for the trouble as this may introduced by our structure change in #4 , please let us know if you have other concerns.

**Comment by protomech** [comment]
@DeepSecurityOpenSourceSupport thank you. I've rebased and removed the extraneous debug.

**Review by DeepSecurityOpenSourceSupport — APPROVED** [review]
Thanks @protomech , the changes looks great.
We're working on the README now, will publish the new release once it is done.

**Pull request merged**